src/main/java/org/kainos/ea/api/ResponsibilityService.java

```diff
@@ -9,16 +9,19 @@
 import java.util.List;
 
 public class ResponsibilityService {
+    private ResponsibilityDao responsibilityDao;
 
-    private ResponsibilityDao responsibilityDao = new ResponsibilityDao();
+    public ResponsibilityService(ResponsibilityDao responsibilityDao) {
+        this.responsibilityDao = responsibilityDao;
+    }
 
     public List<Responsibility> getRoleResponsibilityById(int id) throws FailedToGetRoleResponsibilityException,
```

Comment:
Returning a list of responsibilities here. Providing an id in the method implies your going to fetch one role responsibility by its ID.